src/main/java/http/controllers/UnauthorizedController.java

```diff
@@ -0,0 +1,30 @@
+package http.controllers;
+
+import http.IO.file.InvalidPathException;
+import http.request.HttpRequest;
+import http.response.HttpResponse;
+import http.status.HttpStatus;
+import http.status.InvalidStatusCodeException;
+
+import java.io.IOException;
+import java.util.Base64;
+
+public class UnauthorizedController implements IController {
+
+    private final String authString;
+
+    public UnauthorizedController(String authString) {
+        this.authString = authString;
+    }
+
+    @Override
+    public HttpResponse generateResponse(HttpRequest httpRequest) throws IOException, InvalidPathException, InvalidStatusCodeException {
+        HttpResponse httpResponse = new HttpResponse();
+        httpResponse.setStatus("401");
```

Comment:
Can you change the signature of this method, so that it does not take a String or Int?

Reply:
👍